test/js/computed-style-property-map.js

```diff
@@ -23,7 +23,7 @@ suite('Computed StylePropertyMap', function() {
     assert.strictEqual(propertyStyleValue.cssString, '0.5');
   });
 
-  test('get method returns the first StyleValue in the sequence if a property has been set a sequence ' +
+  test.skip('get method returns the first StyleValue in the sequence if a property has been set a sequence ' +
```

Comment:
Add a TODO about replacing animation-iteration-count with some other sequence-compatible property & enabling this test.